src/app/calendar/page.tsx

```diff
@@ -4,7 +4,7 @@ import React from "react";
 const CalendarPage: React.FC<{}> = () => {
     return (
         <main>
-            <h1>Calendar Page</h1>
+            <h1>Calendar Page </h1>
```

Comment:
can you remove this space? there shouldn't be any changes to `page.tsx` in a component ticket

Reply:
Space removed!